### DISCUSSION TITLE
errore evacuazione seguito da logout

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#670] [Richieste di aiuto]
Purtroppo non solo non appena clicco su evacuazione per compilare il modulo ,compare l'errore 500 , ma effettua automaticamente il logout 

[2024-03-19_18-05.log](https://github.com/iisgiua/giuaschool/files/14654016/2024-03-19_18-05.log)

**Comment by trinko** [comment]
Mi spiace ma non posso esserti di aiuto perché continui a mandarmi dei file di log che non contengono la pagina di errore, ma solo le pagine di Sistema>Manutenzione.

Stai sbagliando qualcosa quando generi i log.

Per generare il log degli errori, seguire la procedura:
- come utente amministratore, andare alla pagina Sistema > Manutenzione e abilitare il debug;
- **eseguire la pagina che causa l'errore e segnarsi l'ora in cui è stata eseguita;**
- tornare alla pagina Sistema > Manutenzione e estrarre il log, **indicando l'ora in cui si è eseguita la pagina di errore**;
- scaricare il file di log e allegarlo al messaggio di segnalazione;
- tornare alla pagina Sistema > Manutenzione e disabilitare il debug.

**Reply by sempliciniluigi89** [reply]
Ho appena eseguito passo dopo passo quanto detto da lei 
[2024-03-19_21-16.log](https://github.com/iisgiua/giuaschool/files/14656080/2024-03-19_21-16.log)

**Comment by trinko** [comment]
Adesso il log è ok.
L'errore è molto strano... devo approfondire meglio.

**Comment by trinko** [comment]
Credo che il problema sia dovuto alla versione PHP che stai utilizzando su Altervista.

Vai al pannello di controllo di Altervista, quindi sul menu **RISORSE**. 
Cerca il riquadro **PHP** e clicca sul collegamento **"Modifica Impostazioni PHP"**.
A questo punto cerca l'impostazione della versione PHP e impostala su **"PHP8"** e clicca sul pulsante **SALVA**.

Riprova quindi per controllare se si ripresenta l'errore.

**Reply by sempliciniluigi89** [reply]
Adesso funziona,grazie mille.

**Comment by ghost** [comment]
@sempliciniluigi89 Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
